app/models/user.rb

```diff
@@ -1,4 +1,8 @@
+# Include default devise modules. Others available are:
 class User < ApplicationRecord
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Similarly, we want to secure users' passwords by hashing them, therefore let's enable the `:database_authenticable` option.